### ISSUE TITLE
qTox portable - folder structure

### ISSUE THREAD

**Issue by ghost** [qTox/qTox#3545]
Hello qtox team!

Thank you for еру nice tool!

At the moment qtox create profile files, when `Make portable` option is chosen, in its exucutable folders. As a result we have a lot of files in this folder. Example with 3 profiles:

<pre>
iconengines
imageformats
libeay32.dll
libgcc_s_sjlj-1.dll
libstdc++-6.dll
libwinpthread-1.dll
OpenAL32.dll
platforms
profile1.db
profile1.ini
profile1.tox
profile2.db
profile2.ini
profile2.tox
profile3.db
profile3.ini
profile3.tox
Qt5Core.dll
Qt5Gui.dll
Qt5Network.dll
Qt5Sql.dll
Qt5Svg.dll
Qt5Widgets.dll
Qt5Xml.dll
qtox.exe
qtox.ini
qtox.log
sqldrivers
ssleay32.dll
</pre>

Each profile have 3 files: .db, .ini and .tox. In case of 10 profiles is not to easy to backup them.

It can be usefully if qtox create profile files in separate folder. This can be folder _tox_ like it is in uTox in portable mode.

Thank you!

**Comment by Deide** [comment]
It would be useful to just store all the profile files in a folder in general. Either `qtox/profiles/...files` or `qtox/profiles/<profile_name>/...files`. Either way, it's better and allows people to back up (or even symlink) persistent data far more easily and programmatically.

**Comment by ghost** [comment]
Good idea i think, having "profiles/<profile_name>/[profile files here]" or at least "profiles/[profile files here]" would be great
